lib/url_parser.js

```diff
@@ -101,6 +101,13 @@ module.exports = function(url, options, callback) {
         record = record[0].join('');
         const parsedRecord = qs.parse(record);
         const items = Object.keys(parsedRecord);
+        if (
```

Comment:
same question here as I have on the other file regarding doing this check here vs L112 vs not at all